frontend/src/components/Account/editProfile.module.css

```diff
@@ -0,0 +1,101 @@
+.profileForm{
+    width: 410px;
+    height: 781.36px;
+    display: flex;
+    background-color: #FFFFFF;
+    border-radius: 6px; 
+    flex-direction: column; 
+    border: 1px solid #D7D7D7;
+    box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.25);
+    box-sizing: border-box; 
+    align-items: center;
+    /* gap: 0.75em; */
+} 
+
+.profileFieldTitles {
+    font-size: 13px;
+} 
+
+.profileFields {
+    border-radius: 6px;
+    border: 1px solid #D7D7D7;
+    width: 363.08px; 
+    height: 46.25px;
+    text-align: center;
+    font-size: 10px;
+    display: flex; 
+    align-items: center;
+}
+
+.editButton, .cancelButton, .saveButton {
+    width: 142.79px; 
+    height: 38.95px; 
+    left: 122px;
+    top: 295px;
+    border-radius: 6px;
+}
+
+.editButton, .saveButton {
+    border: 1px solid #EF476F;
+    background-color: #EF476F;
+    color: white;
+}
+
+.editButton {
+    margin-left: 14em; 
```

Comment:
this large of an `em` is interesting to me, I assume you're doing this to offset it from the text a certain distance away, but maybe some percent value makes more sense? Maybe even a `position` css modifier? No change needed though